commands.go

```diff
@@ -425,9 +476,14 @@ func (c *commandable) GetRange(key string, start, end int64) *StringCmd {
 	return cmd
 }
 
-func (c *commandable) GetSet(key, value string) *StringCmd {
-	cmd := NewStringCmd("GETSET", key, value)
-	c.Process(cmd)
+func (c *commandable) GetSet(key string, value interface{}) *StringCmd {
```

Comment:
Do you mean `GetSet(string, string)` and `GetSetValue(string, interface{})`? Just to save overhead of `interface{}`?